File: src/main/java/com/pythonbyte/spring_into_ai/repository/ChatAuditLogRepository.java
package com.pythonbyte.spring_into_ai.repository;

import com.pythonbyte.spring_into_ai.entity.ChatAuditLog;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ChatAuditLogRepository extends JpaRepository<ChatAuditLog, Long> {
} 

File: src/main/kotlin/com/pythonbyte/spring_into_ai/repositories/ChatAuditLogRepository.kt
package com.pythonbyte.spring_into_ai.repositories

import com.pythonbyte.spring_into_ai.entities.ChatAuditLog
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ChatAuditLogRepository : JpaRepository<ChatAuditLog, Long> 